src/idls/thrift/src/t_yarp_generator.cc

```diff
@@ -1840,6 +1842,22 @@ void t_yarp_generator::generate_struct(t_struct* tstruct)
         yarp_api_keyword = annotations.at("yarp.api.keyword");
     }
 
+    std::string yarp_type_name{};
+    if (annotations.find("yarp.type.name") != annotations.end()) {
+        yarp_type_name = annotations.at("yarp.type.name");
+    }
+    else {
+        yarp_type_name = "yarp/"+name;

```

Comment:
I'm not convinced about this default value, I'd rather enable this only if specified by the user (at least for now), or everyone who defined some type in some other repo will have some "yarp" types.

Reply:
I see. Better `name`? or just an empty string?

Reply:
Empty string and do not add the type part if not defined